setup.py

```diff
@@ -68,7 +68,11 @@ def get_name():
         'tf-explain',
         'slackclient',
         'nest_asyncio',
-        'pycocotools-fix'
+        'pycocotools-fix',
+        'nibabel== 2.5.1',
```

Comment:
is this really only version 2.5.1 or can we do 2.5.1 and above?

Reply:
Changed to 2.5.1 and above